src/snowflake/snowpark/udf.py

```diff
@@ -87,6 +88,7 @@ def __init__(
         packages: Optional[List[Union[str, ModuleType]]] = None,
         _ast: Optional[proto.Udf] = None,
         _ast_id: Optional[int] = None,
+        _session: Optional["snowflake.snowpark.session.Session"] = None,
```

Comment:
I may overlook but I didn't this any caller passing the `_session` parameter?
when is this parameter used

Reply:
Thank you for catching this! Missed passing in the `_session` parameter at the call sites in `UDFRegistration` methods in this file.